examples/qualcomm/utils.py

```diff
@@ -140,7 +140,7 @@ def push(self, inputs=None, input_list=None, files=None):
             for file_name in files:
                 self._adb(["push", file_name, self.workspace])
 
-    def execute(self, custom_runner_cmd=None):
+    def execute(self, custom_runner_cmd=None, method_index=0):
```

Comment:
Is it because we don't have the method name but just method index?

Reply:
I think that's related to the interface we used in `qnn_executor_runner`:
```c++
  const char* method_name = nullptr;
  {
    const auto method_name_result =
        program->get_method_name(FLAGS_method_index);
    ET_CHECK_MSG(method_name_result.ok(), "Program has no methods");
    method_name = *method_name_result;
  }
  ET_LOG(Info, "Using method %s", method_name);
```